examples/Consumer/Program.cs

```diff
@@ -174,16 +192,57 @@ public static void Run_Consume(string brokerList, List<string> topics)
 
                     if (msg.Offset % 5 == 0)
                     {
-                        Console.WriteLine($"Committing offset");
                         var committedOffsets = consumer.CommitAsync(msg).Result;
```

Comment:
Why do a async commit blockingly (waiting for the result)? Why not just do a sync commit directly?

Reply:
there is no sync commit on the api. this is something I'll rectify in 1.0.